src/Altinn.Dan.Plugin.Statensvegvesen/Clients/SvvClient.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Nadobe.Common.Exceptions;
+using Newtonsoft.Json;
+
+namespace Altinn.Dan.Plugin.Statensvegvesen.Clients
+{
+    public class SvvClient
+    {
+        private readonly HttpClient _httpClient;
+
+        public SvvClient(HttpClient httpClient)
+        {
+            _httpClient = httpClient;
+        }
+
+        /// <param name="fodselsnummer">Fødselsnummer på eier, medeier eller leasingtaker</param>
+        /// <returns>Søk gjennomført, responsen inneholder eventuelle treff og/eller responskoder</returns>
+        public async Task<KjoretoysokResponse> SokKjoretoyForFodselsnummer(string fodselsnummer)
+        {
+            if (fodselsnummer == null)
+            {
+                throw new ArgumentNullException(nameof(fodselsnummer));
+            }
+
+            var urlBuilder = new StringBuilder();
+            urlBuilder.Append("?fodselsnummer=").Append(Uri.EscapeDataString(fodselsnummer));
+            var response = await _httpClient.GetAsync(urlBuilder.ToString());
+            try
+            {
+                switch (response.StatusCode)
+                {
+                    case HttpStatusCode.OK:
+                    {
+                        var kjoretoysokResponse = JsonConvert.DeserializeObject<KjoretoysokResponse>(await response.Content.ReadAsStringAsync());
+
+                        return kjoretoysokResponse;
+                    }
+                    case HttpStatusCode.NoContent:
+                    {
+                        throw new EvidenceSourcePermanentClientException(Metadata.ERROR_NO_VEHICLES_FOUND, $"No vehicles found");
+                    }
```

Comment:
The api returns nocontent when no vehicles are found?

Reply:
Yup.
From the spec:
"responses":
          "200": 
            "description": "Søk gjennomført, responsen inneholder eventuelle treff og/eller responskoder"
          "204": 
            "description": "Ingen kjøretøy funnet"
          "400": 
            "description": "Parameterfeil, responsen inneholder kun beskrivelse av feil som responskoder"